experimental/dds/tree2/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -643,35 +619,15 @@ function compareCellPositions(
 	baseMark: Mark<unknown>,
```

Comment:
I notice we don't need `baseMark` aside from its count. I think it might be nice to reduce the API surface of this function by only taking in the count.